test/new-e2e/tests/cws/lib/utils.go

```diff
@@ -0,0 +1,19 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package e2elib
+
+import "math/rand"
+
+const alphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+// RandomString returns a random string of the specified length
```

Comment:
There's perhaps somewhere a util in Agent code that does it, otherwise I don't think this util should be in `cws/lib`.

Reply:
I found a similar function in `pkg/security/utils` but I found it a bit messy to add the agent as a dependency in the  e2e tests module so I reverted back to using uuid-based string generation.